samples/browser/src/main/java/org/mozilla/samples/browser/BrowserFragment.kt

```diff
@@ -69,6 +74,12 @@ class BrowserFragment : Fragment(), BackHandler, DownloadDialogListener {
         downloadsFeature.onNeedToRequestPermissions = { _, _ ->
             requestPermissions(arrayOf(WRITE_EXTERNAL_STORAGE), PERMISSION_WRITE_STORAGE_REQUEST)
         }
+
+        // Accept all DRM content
+        permissionObserver = object : SelectionAwareSessionObserver(components.sessionManager) {
+            override fun onContentPermissionRequested(session: Session, permissionRequest: PermissionRequest): Boolean =
+                permissionRequest.grantIf { it is Permission.ContentProtectedMediaId }
+        }
```

Comment:
Not sure whether we need this in the sample? :)

Reply:
We don't, it was mostly for testing and showing the API. Also, Gecko doesn't send this permission request anyway. It's a setting in about:config whether or not DRM / EME content is allowed.